include/mathfu/vector.h

```diff
@@ -410,6 +410,25 @@ class Vector {
     MATHFU_VECTOR_OPERATION(vector->data[i] = data_[i]);
   }
 
+  /// @brief Compare 2 Vectors of the same size
+  ///
+  /// @return True if the 2 vectors contains the same value
```

Comment:
Do you think it's worth mentioning in the docs that if two vectors have floating point values it the likelyhood of v1 == v2 is very small and it would be better for the use to do "v1.LengthSquared(v2) < epsilon" instead?